test/fixtures_s3_test.go

```diff
@@ -0,0 +1,148 @@
+//+build integration fixtures
+
+package test_test
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/fixtures"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/applike/gosoline/pkg/test"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"testing"
+)
+
+const (
+	BucketName    = "s3-fixtures-test-bucket"
+	BasePath      = "test_data/s3_fixtures_test_data"
+	BasePathPurge = "test_data/s3_fixtures_purge_test_data"
+)
+
+type FixturesS3Suite struct {
+	test.Suite
+
```

Comment:
no empty lines in struct definitions please

Reply:
noted and applied. Although I found multiple structs having empty lines separating attributes. Is it not a good idea to separate wrapped interfaces, structs (inside a struct), exported attributes from private attributes by an empty line?